package.json

```diff
@@ -19,9 +19,11 @@
     "umd"
   ],
   "dependencies": {
-    "amigo2-instance-data": "^2.5.7",
+    "amigo2-instance-data": "^2.5.11",
```

Comment:
Probably better to use exact version `2.5.11` instead of compatible version `^2.5.11`, in case of problem caused by minor incompatibility.

Reply:
I'm leaving it this way for now.    Links get fixed and added with changes upstream, so we need to make sure to get those.